akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -1965,9 +1955,9 @@ trait FlowOps[+Out, +Mat] {
    */
   def async: Repr[Out]
 
-  /** INTERNAL API */
-  private[scaladsl] def andThen[T](op: SymbolicStage[Out, T]): Repr[T] =
-    via(SymbolicGraphStage(op))
+  //  /** INTERNAL API */
+  //  private[scaladsl] def andThen[T](op: SymbolicStage[Out, T]): Repr[T] =
+  //    via(SymbolicGraphStage(op))
```

Comment:
Ouch, seems I missed some - I first commented out to then delete, will skim through again to delete all remaining